go/vt/schema/parser_test.go

```diff
@@ -40,14 +42,44 @@ func TestParseAlterTableOptions(t *testing.T) {
 	}
 	for query, expect := range tests {
 		schema, table, options := ParseAlterTableOptions(query)
-		if schema != expect.schema {
-			t.Errorf("schema: %+v, want:%+v", schema, expect.schema)
-		}
-		if table != expect.table {
-			t.Errorf("table: %+v, want:%+v", table, expect.table)
-		}
-		if options != expect.options {
-			t.Errorf("options: %+v, want:%+v", options, expect.options)
+		assert.Equal(t, expect.schema, schema)
+		assert.Equal(t, expect.table, table)
+		assert.Equal(t, expect.options, options)
+	}
+}
+
+func TestNormalizeOnlineDDL(t *testing.T) {
+	type expect struct {
+		sqls    []string
+		isError bool
+	}
+	tests := map[string]expect{
+		"alter table t add column i int, drop column d": {sqls: []string{"alter table t add column i int, drop column d"}},
+		"create table t(id int primary key)":            {sqls: []string{"create table t(id int primary key)"}},
+		"drop table t":                                  {sqls: []string{"drop table t"}},
+		"drop table if exists t":                        {sqls: []string{"drop table if exists t"}},
+		"drop table t1, t2, t3":                         {sqls: []string{"drop table t1", "drop table t2", "drop table t3"}},
+		"drop table if exists t1, t2, t3":               {sqls: []string{"drop table if exists t1", "drop table if exists t2", "drop table if exists t3"}},
+		"create index i_idx on t(id)":                   {sqls: []string{"alter table t add index i_idx (id)"}},
+		"create index i_idx on t(name(12))":             {sqls: []string{"alter table t add index i_idx (`name`(12))"}},
+		"create index i_idx on t(id, `ts`, name(12))":   {sqls: []string{"alter table t add index i_idx (id, ts, `name`(12))"}},
+		"create unique index i_idx on t(id)":            {sqls: []string{"alter table t add unique index i_idx (id)"}},
+		"create index i_idx using btree on t(id)":       {sqls: []string{"alter table t add index i_idx using btree (id)"}},
+		"create index with syntax error i_idx on t(id)": {isError: true},
+		"select * from t":                               {isError: true},
+		"drop database t":                               {isError: true},
+	}
+	for query, expect := range tests {
+		normalized, err := NormalizeOnlineDDL(query)
```

Comment:
nit:
```golang
for query, expect := range tests {
	t.Run(query, func(t *testing.T) {
...
	}
}
})
```

sub tests makes it look good in test output

Reply:
TIL actually

Reply:
fixed